Translation tips reworked (#90)
monero-ecosystem/monero-translations: translation-tips.md

- [Documentation and website for Kovri (anonymous router)](https://github.com/monero-ecosystem/monero-translations/blob/master/translate-kovri-website.md)
- [Monerujo (Android wallet)](https://github.com/monero-ecosystem/monero-translations/blob/master/translate-monerujo.md)

Other Monero-related projects, created by community members, are to be found in the [Monero Ecosystem](https://github.com/monero-ecosystem/meta).

> We have a localization platform called Weblate, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it only supports the website, Community, GUI and CLI wallet, but will soon also be available for Monerujo, the Android wallet.

Reviewer: ```suggestion
We have a localization platform called Weblate, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it supports the website, community material, GUI and CLI wallet.

```

There are no plans to include Monerujo at this time, so that part can be removed.

For now, the default workflow of all projects currently on Weblate, is entering the translations in a text editor and making a pull request in Git or on GitHub to upload your work.

The TS files containing source text and translation strings for the actual software (Monero GUI, CLI and daemon) can be translated in [Qt Linguist](http://doc.qt.io/archives/qt-4.8/linguist-translators.html), which offers some limited computer-assisted translation (CAT) features, like copying repeated strings. You won't have to worry about keeping the XML tags intact. If you instead use a text editor, you'll have to remove the label `type="unfinished"` manually after translating each string.

![GUI version 0.12 in Dutch](media/translation-tips/gui.png)

## 2. People

> The current system is a marriage of language experts, and the software inclined. Don't hesitate to ask our localization coordinator @erciccione for technical help and planning advice. You can reach him on the #monero-translations channel on IRC (freenode), [on Reddit](https://www.reddit.com/user/ErCiccione/) or by e-mail at translate[at]getmonero[dot]org.

Reviewer: ```suggestion
The current system is a marriage of language experts, and the software inclined. Don't hesitate to ask to the members of the Localization Workgroup for technical help and planning advice. You find a list of contacts on the README of this repository.
```

Better avoid personal references and refer to the workgroup instead.

Other Monero-related projects, created by community members, are to be found in the [Monero Ecosystem](https://github.com/monero-ecosystem/meta).

We have a localization platform called Weblate, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it supports the website, community material, GUI and CLI wallet.

For now, the default workflow of all projects currently on Weblate, is entering the translations in a text editor and making a pull request in Git or on GitHub to upload your work.

The TS files containing source text and translation strings for the actual software (Monero GUI, CLI and daemon) can be translated in [Qt Linguist](http://doc.qt.io/archives/qt-4.8/linguist-translators.html), which offers some limited computer-assisted translation (CAT) features, like copying repeated strings. You won't have to worry about keeping the XML tags intact. If you instead use a text editor, you'll have to remove the label `type="unfinished"` manually after translating each string.

![GUI version 0.12 in Dutch](media/translation-tips/gui.png)

## 2. People

> The current system is a marriage of language experts, and the software inclined. Don't hesitate to ask to the members of the Localization Workgroup for technical help and planning advice. You find a list of contacts on the README of this repository.

Author: ```suggestion
The current system is a marriage of language experts, and the software inclined. Don't hesitate to ask to the members of the Localization Workgroup for technical help and planning advice. You find a list of contacts in the README of this repository.

```

Consistency is even more important than choosing the best translation. For example, a user who wants to know what the view key does, needs to find the same term that's used in Monero wallets on the Monero website.

Before using dictionaries (see paragraph 7 below), don't forget to search in projects that have already been translated. For example, if you're adding translations to the GUI, look for the existing translations in the monero-core repository, or in other repositories like the getmonero.org website if they have been translated. Or, [use a phrase book in QtLinguist](http://doc.qt.io/qt-4.8/linguist-translators.html#phrase-books).

> To help different translators use the same terminology, lists of terms are shared [on Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/home). At the moment (Oct 2020), glossaries are available for Swedish, Italian, German and Spanish.

Reviewer: Glossaries are now here: https://github.com/monero-ecosystem/monero-translations#terminology-guides

> To help different translators use the same terminology, lists of terms are shared [on Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/home). At the moment (Oct 2020), glossaries are available for Swedish, Italian, German and Spanish.

Author: ```suggestion
To help different translators use the same terminology, lists of terms are shared [on GitHub](https://github.com/monero-ecosystem/monero-translations#terminology-guides). At the moment (Oct 2020), glossaries are available for Brazillian Portugese, French, Swedish, Simplified Chinese, Italian, German and Spanish.

```

The best practice would be to compile and test the translated version of the software before submitting a pull request. If you're not able to do this, maybe you can find someone who can?

Actually use your translations. Please don't think: "I don't need it, I speak English."

> Translations are reviewed by ErCiccione and volunteers on GitHub before being merged. If you don't have time to translate, please consider reviewing the translations of others.

Reviewer: ```suggestion
Translations are reviewed by volunteers on GitHub and Weblate before being merged. If you don't have time to translate, please consider reviewing the translations of others.

```
